testsuite/api/test_omsExport.py

```diff
@@ -57,6 +57,7 @@ def printStatus(status, expected):
 ## Result:
 ## status:  [correct] ok
 ## status:  [correct] ok
+## warning: [oms_setSignalFilter] is deprecated , use [oms_addSignalsToResults] to add signals and [oms_removeSignalsFromResults] to remove signals
```

Comment:
remove the white-space before the comma

Reply:
done with this https://github.com/OpenModelica/OMSimulator/pull/915/commits/ae8174960b62f59c22f92e87c336328bd0194111